plugins/azure-devops/README.md

```diff
@@ -16,6 +16,12 @@ Lists the top _n_ Active, Completed, or Abandoned Pull Requests for a given repo
 
 ![Azure Repos Pull Requests Example](./docs/azure-devops-pull-requests.png)
 
+### Azure Git Tags
+
+Lists all Git Tags for a given repository
+
+![Azure Repos Pull Requests Example](./docs/azure-devops-git-tags.png)
```

Comment:
"Pull Requests" should be changed to "Git Tags"

Reply:
Thanks for the feedback, and sorry for the whoopsies! I've pushed changes. Let me know if this resolves your comments to your satisfaction!